Makefile

```
@@ -42,10 +42,17 @@ bin/dep-${DEP_VERSION}:
 	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | INSTALL_DIRECTORY=bin DEP_RELEASE_TAG=v${DEP_VERSION} sh
 	@mv bin/dep $@
 
+bin/packr2:
+	go get -u github.com/gobuffalo/packr/v2/packr2
+
 .PHONY: vendor
-vendor: bin/dep ## Install dependencies
+vendor: bin/dep bin/packr2 ## Install dependencies
 	bin/dep ensure -v -vendor-only
 
+.PHONY: pre-build ## Pre build bundles of static assets
+pre-build:
+	@cd internal/cli/command/form && packr2
```

```suggestion
	cd internal/cli/command/form && $(abspath bin/packr2)
```

do not hide the command, and do not assume that current gopath/bin is in the path (untested suggestion)